pkg/namesgenerator/names-generator.go

```diff
@@ -358,6 +358,9 @@ var (
 	rnd = rand.New(random.NewSource())
 )
 
+// GetRandomName generates a random name from the list of adjectives and surnames in this package
+// formatted as "adjective_surname". For example 'focused_turing'i. If retry is non-zero, a random
```

Comment:
Just wondering, it "names-generator" should have the "-" removed.  @LK4D4 @cpuguy83 ?